### ISSUE TITLE
Login in pop up redirects to account page

### ISSUE THREAD

**Issue by dhritzkiv** [paypal/paypal-checkout#688]
### Description

When the PayPal popup is displayed on the host page (due to pop ups being blocked, since it's in a webview), the popup iframe will redirect the host page after the user logs into PayPal.

This doesn't seem to affect /every/ browser, or PayPal user, but it consistently affects the same browser builds / PayPal users.

- This happens in both sandbox, and in production.
- PayPal version 4.0.202
- Can't link to a page at the moment, as it's not public yet, but will soon.

Here is the link that the popup iframe was trying to redirect the host page to, after logging into PayPal: https://www.paypal.com/webapps/hermes?buttonSessionID=b1adcf7895_ge4tumjuhiyts&country.x=CA&env=production&flow=1-P&fundingSource=paypal&locale.x=en_US&logLevel=warn&sessionID=43bf96f026_ge4tumjuhiyts&token=EC-68100368XG8756212&uid=0af3464955&ulClientRedirect=timeout&version=4&xcomponent=1.

Seems to be similar to #423 
Possibly related to #677

### Affected browsers

- [x] IE (noticed in IE11 on Windows 7, could affect others)
- [x] Web View / Safari ViewController (noticed in Safari 9 on macOS Sierra)

**Comment by bluepnume** [comment]
Do you have the user agent of the web view? If so we can disable popups for it and default to an iframe.

For IE11, do you still see the issue? And do you have any kind of intranet, protected, trusted mode enabled? IE can be really finicky with any of these settings.

**Comment by dhritzkiv** [comment]
On Windows, the user agent is the default IE 11 web view user agent with a custom string appended to it: `Mozilla/5.0 (Windows NT 6.2; Win64; x64; Trident/7.0; rv:11.0) like Gecko; [MFStudio; Version/1.0.93; Android; Version/0.0]`

We added `Android; Version/0.0` to force faster iframe behaviour (without it, the popup window behaviour is still blocked, but it takes a longer while for the iframe to activate). Ideally, it'd be nice to be able to add `iframe: true` (or similar) to the config to disable pop up window behaviour, but that's a separate issue.

Anyway, we haven't been able to test this bug lately, as we disabled PayPal integration completely. I might get a chance to test it later this month when I have some time to re-enable it.
